extensions/crypto/Cargo.toml

```diff
@@ -18,5 +18,11 @@ deno_core = { version = "0.92.0", path = "../../core" }
 deno_web = { version = "0.41.1", path = "../web" }
 tokio = { version = "1.7.1", features = ["full"] }
 rand = "0.8.3"
-ring = "0.16.20"
+ring = { version = "0.16.20", features = ["std"] }
+rsa = "0.4.0" # TODO: remove "pem" feature when next release is on crates.io
```

Comment:
This TODO is outdated

Reply:
No, I am waiting for https://github.com/RustCrypto/RSA/commit/366ff6eac3180be10362855564b212987aa73030 to land in a release.